src/main/java/com/minecolonies/commands/KillCitizen.java

```diff
@@ -0,0 +1,130 @@
+package com.minecolonies.commands;
+
+import com.minecolonies.colony.CitizenData;
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.entity.EntityCitizen;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.DamageSource;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.TextComponentString;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * List all colonies.
+ */
+public class KillCitizen extends AbstractSingleCommand
+{
+
+    private static final String ID_TEXT                         = "§2ID: §f";
+    private static final String NAME_TEXT                       = "§2 Name: §f";
+    private static final String COORDINATES_TEXT                = "§2Coordinates: §f";
+    private static final String REMOVED_MESSAGE                 = "Has been removed";
+    private static final String NO_COLONY_CITIZEN_FOUND_MESSAGE = "No citizen %d found in colony %d.";
+    private static final String COORDINATES_XYZ                 = "§4x=§f%s §4y=§f%s §4z=§f%s";
+
+    public static final String DESC                             = "kill";
+
+    /**
+     * Initialize this SubCommand with it's parents.
+     *
+     * @param parents an array of all the parents.
+     */
+    public KillCitizen(@NotNull final String... parents)
+    {
+        super(parents);
+    }
+
+    @NotNull
+    @Override
+    public String getCommandUsage(@NotNull final ICommandSender sender)
+    {
+        return super.getCommandUsage(sender) + "";
+    }
+
+    @Override
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
+    {
+        final int colonyId = getIthArgument(args, 0, -1);
+        final int citizenId = getIthArgument(args, 1, -1);
```

Comment:
We could allow citizen names as autocomplete options, then make them autocomplete on tab.
But maybe for later...